source/DomainTheory/BasesAndContinuity/Continuity.lagda

```diff
@@ -506,3 +506,50 @@ module _
   ∥∥-functor structural-continuity-of-dcpo-preserved-by-continuous-retract
 
 \end{code}
+
+Added 8 July 2024.
+
+The purpose of the following construction is to show that structural continuity
```

Comment:
Did we decide to still use the terminology "structural continuity" rather than "continuity data"?